spec/support/test_helpers/cycle_timetable_helper.rb

```diff
@@ -15,12 +15,3 @@ def after_apply_reopens
     CycleTimetable.apply_reopens(2021) + 1.day
   end
 end
-
-module RecruitmentCycle
-  CYCLES = {
-    '2022' => '2021 to 2022',
-    '2021' => '2020 to 2021',
-    '2020' => '2019 to 2020',
-    '2019' => '2018 to 2019',
-  }.freeze
-end
```

Comment:
I had to remove this on the spec support because my tests were failing even after I did the fix because of this monkey patch.

Reply:
I guess it makes sense to be able to stub this in the tests, but is it not possible to do something slightly less hacky than monkey-patching? eg `allow(RecruitmentCycle).to receive(:CYCLES).and_return(...)`?

Reply:
Yes, I agree. But then should I add a global `allow` in a` before(:all)` block, in rspec for integration/system tests? Because the system specs are relying on 2019 (one problem leading to another 😆).

Reply:
how about implementing it in a method (`stub_recruitment_cycles` or some such) and then calling it in the `before` blocks of whatever specs rely on it.

Reply:
Only 4 specs relying on it. I thought that we will be so much more. I'll comeback to this PR next Monday while I finish references.

Reply:
There are 47 examples failing 😞. I don't know if I should stub const in each one of them. The main problem is that we have many specs relying on 2019. If we fix this problem this stub const is not needed. Although to fix that is also a big work. I do wonder if I should put this back in and create a card to tackle the problem